Formula/mesa.rb

```diff
@@ -93,7 +93,6 @@ def install
         args << "-Dopengl=true"
         args << "-Dgles1=true"
         args << "-Dgles2=true"
-        args << "-Dxvmc=true"
```

Comment:
Do we want to make this `-Dgallium-xvmc=false`? Also, could you squash your commits together please? Thanks!

Reply:
Having a more detailed look at the "meson.build" file of "mesa v21.2.4" library found the following:
`
_xvmc = get_option('gallium-xvmc')
if _xvmc == 'true'
  _xvmc = 'enabled'
  warning('gallium-xvmc option "true" deprecated, please use "enabled" instead.')
elif _xvmc == 'false'
  _xvmc = 'disabled'
  warning('gallium-xvmc option "false" deprecated, please use "disabled" instead.')
endif
`
So first of all when fixing this bug here, we should use the keys "enabled" or "disabled" instead of "true" and false". Having a further look into the file I would agree with your suggestion, that if we want to turn it off we need to explicitly setting the option to "disabled", as the default value is "enabled". Going to repeat a test having the option set to "disabled" explicitly.

Reply:
Great; thanks!

Reply:
Explicitly setting the value to "disabled" caused no trouble on my machine as well. The Option's related name and value has been updated and the commits have been squatshed as requested